packages/@o3r/create/src/index.ts

```diff
@@ -119,8 +120,26 @@ const addOtterCore = (relativeDirectory = '.') => {
   }
 };
 
+/**
+ * When the monorepo is created, the package json will keep only the ng script
+ * @param relativeDirectory package.json directory
+ */
+const cleanPackageJsonScripts = (relativeDirectory = '.') => {
+  const packageJsonPath = resolve(process.cwd(), relativeDirectory, 'package.json');
+  try {
+    const packageJsonContent = JSON.parse(readFileSync(packageJsonPath, { encoding: 'utf-8' })) as PackageJson;
+    packageJsonContent.scripts = { ng: 'ng' };
+    writeFileSync(packageJsonPath, JSON.stringify(packageJsonContent, null, 2));
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error(error);
+    process.exit(2);
```

Comment:
```suggestion
    process.exit(3);
```
2 is already use line 119

Reply:
the modification on this file was fully discarded